book/website/reproducible-research/rdm/rdm-spreadsheets.md

```diff
@@ -6,7 +6,14 @@ There is a collection of [horror-stories](http://www.eusprig.org/horror-stories.
 editing processes.
 Some of these mishaps are not unique to spreadsheets, but [many are](https://doi.org/10.1186/s13059-016-104).
 
-Most problems can be avoided with two rules: use spreadsheet in a text only format (.csv or .tsv), avoid manipulating and analysing data in spreadsheet software (this includes copy-paste). Spreadsheets are a powerful tool, if one recognise that computers are not fancy typewriters, but that the data has to be organised in a specific form to be usable by computers (and data scientist).
+Most problems can be avoided with the three rules explained below: 
+- use spreadsheet in a text only format (.csv or .tsv)
+- create tidy spreadsheets
```

Comment:
```suggestion
- create tidy and consistent spreadsheets
```

Reply:
What do you mean exactly with "consistent" ? This would need to be explained in the detailed description.

Reply:
Does it refer to different spreadsheet being consistent one with the others? (same unit used, same header names,...)

Reply:
google gives this in the first page : http://www.ssrb.org/resources/standards/ce/sma3/sheet_consistency, which is full of very, very bad advices...

Reply:
Gees, consistent in the sense - if you have used a certain delimiter, stick with it in your data set. If you have a certain placeholder for missing data, stick with it. (and so on). What would that be called (if not consistent)? Any suggestion?

Reply:
delimiters is indeed included in "using .csv format". The other elements are, I think, included in the "Accepted values" section. Do you think we should had some information there, and do you think this should go into the main 3 rules (it would then be 4) ?

Reply:
I will close this without commit and work the text later